src_pop/core/biogeochem/POP.F90

```diff
@@ -2007,14 +2048,23 @@ END SUBROUTINE Patch_partial_disturb2
 
   !*******************************************************************************
 
-
+  !! # Patch Disturbance
```

Comment:
Do you want this comment in the in line documentation?

Reply:
I have replaced all "!****" comments with "!----"